lib/request/rack.rb

```diff
@@ -136,6 +136,7 @@ def if_modified_since
     accessor(:host,         Key.new('SERVER_NAME'))
     accessor(:query_string, Key.new('QUERY_STRING'))
     accessor(:content_type, Key.new('CONTENT_TYPE'), nil)
+    accessor(:cookies,      Key.new('HTTP_COOKIE'), EMPTY_STRING)
```

Comment:
If I'm not mistaken this would involve doing all the checks outlined in [RFC6265](https://tools.ietf.org/html/rfc6265#section-4.2) and I'm not really up to that anytime soon.

FWIW,  the only things currently being done to "parse" `Cookie` headers are:
  1. https://github.com/snusnu/cookie/blob/master/lib/cookie/registry.rb#L18
  2. https://github.com/snusnu/cookie/blob/master/lib/cookie.rb#L56

If anything, I'd consider any kind of cookie validation outside the scope of this specific PR.